Bottom: a79a71624a0e34ac5baaba8b5311c5067f757077
Top:    54a6a18124ccc9d0c34aac21e0abc9ad3916b36a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-10-12 17:29:26 +0100

Refresh of increase-initial-heap-size-fro

---

diff --git a/src/heap.c b/src/heap.c
index e889d7e..4959b58 100644
--- a/src/heap.c
+++ b/src/heap.c
@@ -165,7 +165,7 @@ heap_insert (heap_t * heap, cost_t cost, void *data)
     {
       heap->max *= 2;
       if (heap->max == 0)
-	heap->max = 512;		/* default initial heap size */
+	heap->max = 256;		/* default initial heap size */
       heap->element =
 	realloc (heap->element, heap->max * sizeof (*heap->element));
     }
